README.md

```diff
@@ -117,4 +117,214 @@ containers:
                name: test-atlas-operator-project-test-cluster-theuser
                key: connectionStringStandardSrv
 
-```
\ No newline at end of file
+```
+### Step 4. Test Database as a Service (DBaaS) on OpenShift
+
+The Atlas Operator is integrated with the [Red Hat Database-as-a-Service (DBaaS) Operator](https://github.com/RHEcosystemAppEng/dbaas-operator) which allows application developers to import database instances and connect to the databases through the [Service Binding Operator](https://github.com/redhat-developer/service-binding-operator). More information can be found [here](https://github.com/RHEcosystemAppEng/dbaas-operator#readme).
+
+Note that both the DBaaS Operator and Atlas Operator should be installed through the [Operator Lifecyle Manager (OLM)](https://github.com/operator-framework/operator-lifecycle-manager).
+
+**1.** Check DBaaS Registration
+
+If the DBaaS Operator has been deployed in the OpenShift Cluster, the Atlas Operator automatically creates a cluster level [DBaaSProvider](https://github.com/RHEcosystemAppEng/dbaas-operator/blob/main/config/crd/bases/dbaas.redhat.com_dbaasproviders.yaml) customer resource (CR) object `mongodb-atlas-registration` to automatically register itself with the DBaaS Operator. 
+
+```
+apiVersion: dbaas.redhat.com/v1alpha1
+kind: DBaaSProvider
+metadata:
+  labels:
+    related-to: dbaas-operator
+    type: dbaas-provider-registration
+  name: mongodb-atlas-registration
+spec:
+  connectionKind: MongoDBAtlasConnection
+  credentialFields:
+  - displayName: Organization ID
+    key: orgId
+    required: true
+    type: string
+  - displayName: Public API Key
+    key: publicApiKey
+    required: true
+    type: string
+  - displayName: Private API Key
+    key: privateApiKey
+    required: true
+    type: maskedstring
+  inventoryKind: MongoDBAtlasInventory
+  provider:
+    displayDescription: Cloud-hosted MongoDB service on AWS, Azure and Google Cloud
+    displayName: MongoDB Atlas Cloud Database Service
+    icon:
+      base64data: <icon image data>
+      mediatype: image/png
+    name: Red Hat DBaaS / MongoDB Atlas
+```
+If the Atlas Operator is undeployed with the OLM, the above registration CR gets cleaned up automatically.
+
+**2.** Check MongoDBAtlasInventory Customer Resource
+
+When an application developer creates a [DBaaSInventory](https://github.com/RHEcosystemAppEng/dbaas-operator/blob/main/config/crd/bases/dbaas.redhat.com_dbaasinventories.yaml) CR for MongoDB, the DBaaS Operator creates a MongoDBAtlasInventory CR, and the Atlas Operator discovers the clusters and  instances, and sets the result in the CR status. 
```

Comment:
this would be an administrator instead of an application developer

Reply:
OK